pkg/collector/corechecks/cluster/orchestrator/processors/processor.go

```diff
@@ -160,36 +162,37 @@ func (p *Processor) Process(ctx *ProcessorContext, list interface{}) (processRes
 		resourceMetadataModels = append(resourceMetadataModels, resourceMetadataModel)
 
 		// Add resource manifest
-		resourceManifestModels = append(resourceManifestModels, &model.Manifest{
+		manifest := &model.Manifest{
 			Type:            int32(ctx.NodeType),
 			Uid:             string(resourceUID),
 			ResourceVersion: resourceVersion,
 			Content:         yaml,
 			Version:         "v1",
 			ContentType:     "json",
-		})
+		}
+		if ctx.BufferedManifestEnabled {
+			ctx.ManifestChan <- manifest
```

Comment:
open for different opinions, but if we align this with the check below we could safe that `if` check here and only need it once below

Reply:
Thanks for the review, I move this part to the check below